templates/tfengine/recipes/org/project.yaml

```diff
@@ -12,14 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-{{$output_path := get . "OUTPUT_PATH" (printf "./project.%s" .PROJECT_ID)}}
+{{$output_path := get . "OUTPUT_PATH" .PROJECT.PROJECT_ID}}
 
 templates:
 - component_path: "../../components/terragrunt/leaf"
   output_path: "{{$output_path}}/project"
 - name: "project"
   component_path: "../../components/project/project"
   output_path: "{{$output_path}}/project"
+  flatten:
```

Comment:
why do we flatten the `PROJECT` here but not in other recipes?

Reply:
None of the other templates in this list require the fields within PROJECT (e.g. they get the project ID through a terragrunt reference), so there was no need to pass in PROJECT keys to them